code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -772,3 +772,9 @@
 	id = /datum/reagent/mutationtoxin/plasma
 	results = list(/datum/reagent/mutationtoxin/plasma = 5)
 	required_reagents  = list(/datum/reagent/aslimetoxin = 5, /datum/reagent/toxin/plasma = 60, /datum/reagent/uranium = 20)
+
+/datum/chemical_reaction/shiftium
+	name = "Shiftium"
+	id = "shiftium"
+	results = list(/datum/reagent/shiftium = 10)
+	required_reagents = list(/datum/reagent/toxin/plasma = 25, /datum/reagent/liquidgibs = 25, /datum/reagent/nitrous_oxide = 10)
```

Comment:
This should probably have a more diverse (harder to make) required reagent list rather than having a high initial volume requirement

Reply:
What do you suggest?

Reply:
A stimulant maybe? Meth comes to mind but that might make Shiftium a bit _too_ complicated to make. You could replace the liquid gibs with something like Aranesp + fenatyl, up to you really. It's just a bit too simple in it's current state

Reply:
Well, I intended it to be a reagent that is derived from changeling blood/plasma/matter mixed with something. This changeling plasma/blood/whatever would spawn from changeling gibs (headcrab or ground down meat/toma meat) so it would be a bit more complicated to obtain. Hence the liquid gibs.

I didn't implement it because I wasn't sure it was a good idea that would have worked.

Reply:
liquid gibs are pretty easy to obtain, you can get a bunch by running around maints looking for a dead rat. This recipe is only challenging to make for those who can't find liquid gibs, otherwise it's pretty simple.

Reply:
Here's an idea: I made a push.

You can only get it from cling gibs that you can get from changelings (or teratomas)

Reply:
Ooo, cool idea but how will sec/chem be able to test a ling if no other lings have died? Assuming they can't get incinerated first

Reply:
> Ooo, cool idea but how will sec/chem be able to test a ling if no other lings have died? Assuming they can't get incinerated first

Cargo?

Reply:
fair, but 20,000 can be a pretty big hurtle without gas mining.